website/src/documentation/transforms/java/aggregation/hllcount.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: section
+title: "HllCount"
+permalink: /documentation/transforms/java/aggregation/hllcount/
+section_menu: section-menu/documentation.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Latest
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/extensions/zetasketch/HllCount.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br>
+
+Estimate the number of distinct elements using the HyperLogLog++ algorithm.
+Transforms to create, merge, and extract from sketches:
+
+* `HllCount.Init` aggregates inputs into HLL++ sketches.
+* `HllCount.MergePartial` merges HLL++ sketches into a new sketch.
+* `HllCount.Extract` extracts the estimated count of distinct elements from HLL++ sketches.
+
+## Examples
+**Example 1**: create long-type sketch for a `PCollection<Long>` and specify precision
```

Comment:
Would say: "Create [a] long-type sketch... and specify a [custom] precision:" 
(same below, and would end the example descriptions with ":") 
+ maybe be a bit more verbose on the sketch: e.g., "Create a long-type sketch for ... , which can be stored and can be merged with other sketches or from which count distinct estimates can be extracted later"? (if yes, I would also move the custom precision to Example 2).

Reply:
SGTM. Done!